extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -21,6 +21,15 @@ export class AmpCarouscroll extends BaseCarousel {
   }
   /** @override */
   buildCarousel() {
+    this.element.classList.add('-amp-slidescroll');
+
+    /** @private {!Array<!Element>} */
+    this.slides_ = this.getRealChildren();
+
+    this.slides_.forEach(slide => {
```

Comment:
I'd like to explore an option where we'd reparent each slide in its own container with `display:flex` that would center content. Doing `top:50%; transform(-50%)` is not very flexible.